README.md

```diff
@@ -1,3 +1,5 @@
-# Northcoders News API
+# Initial requirements
+
+In order to connect to the databases locally 2 .env files are required; .env.test & .env.development. They must contaim PGDATABASE=nc_news_test and PGDATABASE=nc_news respectively.
+
```

Comment:
Good, don't forget to tell them to npm i dotenv

Reply:
Noted